src/AuthModule/Components/Login/Login.tsx

```diff
@@ -1,10 +1,169 @@
-import React from "react";
-
+import { useForm } from "react-hook-form";
+import logo from "../../../assets/PMS 3.svg";
+import React, { useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import axios from "axios";
+import { toast } from "react-toastify";
+import Loader from "../../../SharedMoudule/Components/Loading/Loading";
 export default function Login() {
-  
+  const [showPassword, setShowPassword] = useState(true);
+  const [isLoading, setIsLoading] = useState(false);
+  const navigate = useNavigate()
+
+
+async function apiLogin(values) {
```

Comment:
Hey team, while your code looks great, let's consider giving a more descriptive name to the 'apiLogin' function to make its purpose clearer. 😊 How about renaming it to one of the following options:
1. authenticateUser 🛡️: Clearly indicates its role in authenticating users.
2. loginUser 💼: Describes its function of handling user login actions.
3. handleLogin 🚪: Highlights its role in managing the login process.
4. performLoginRequest 📡: Clearly communicates its action of making a login request to the API.
5. processLogin ⚙️: Concisely describes its function in managing the login process.
Choose the one that resonates best with your code style and purpose. Keep up the great work! 👍🚀